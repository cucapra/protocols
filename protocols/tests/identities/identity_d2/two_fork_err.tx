// ARGS: --verilog identities/identity_d2/identity_d2.v --protocol identities/identity_d2/identity_d2.prot
// RETURN: 101
two_fork_err(1, 1);
