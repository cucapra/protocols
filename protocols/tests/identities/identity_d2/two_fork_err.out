error: Thread 0 (transaction 'two_fork_err') attempted to fork more than once
   ┌─ identities/identity_d2/identity_d2.prot:21:3
   │
21 │   fork();
   │   ^^^^^^^ first fork() called here
   ·
24 │   fork();
   │   ^^^^^^^ second fork() called here

